backend/apid/graphql/handler_filters.go

```diff
@@ -0,0 +1,23 @@
+package graphql
+
+import (
+	v2 "github.com/sensu/sensu-go/api/core/v2"
+	"github.com/sensu/sensu-go/backend/apid/graphql/filter"
+)
+
+// HandlerFilters returns collection of filters used for matching resources.
+func HandlerFilters() map[string]filter.Filter {
+	filters := map[string]filter.Filter{
+		// type:pipe | type:socket | type:set
```

Comment:
I'm not sure `type:socket` is a correct example. In the docs, I think we distinguish between `udp` and `tcp` connections ([src](https://docs.sensu.io/sensu-go/5.7/reference/handlers/#spec-attributes)).

I might be missing something, but is there logic that takes the `type:socket` filter and returns `udp` and `tcp` handlers?